### DISCUSSION TITLE
Camunda - How to delete the bank instace_id history by api or select

### DISCUSSION THREAD

**Discussion by Mendesz02** [camunda-community-hub/community#138] [General]
good morning

Normally when I have many instances in my cockpit I stop them all through postman, however the history still remains and I currently have 100gb of history alone.

How can I delete them by select or api?

I need to clean these tables: 
hi_details
hi_actinst
hi_varinst

att,

**Comment by xomiamoore** [comment]
Hey @Mendesz02, this community is for extensions and plugins in the Camunda Community Hub organization. For other Camunda questions, please post on the forum here - https://forum.camunda.io/
